tools/sizes.py

```diff
@@ -158,14 +158,14 @@ def main():
     for group, (segments, total) in sizes:
         number, used, segments = filter_segments(segments)
 
-        print(f". {group:<8}, used {used} / {total} bytes ({percentage(used, total)})")
-        print("|   SEGMENT  BYTES    DESCRIPTION")
+        sys.stderr.write(f". {group:<8}, used {used} / {total} bytes ({percentage(used, total)})\n")
+        sys.stderr.write("|   SEGMENT  BYTES    DESCRIPTION\n")
```

Comment:
`sys.stdout = sys.stderr` just before print() will effectively do the same thing